jinahub/encoders/image/BigTransferEncoder/tests/unit/test_big_transfer.py

```diff
@@ -102,3 +100,19 @@ def test_encoding_override_chunks():
     assert doc.embedding is None
     for i in range(3):
         assert doc.chunks[i].embedding.shape == (2048,)
+
+
+@pytest.mark.gpu
+def test_encoding_gpu():
+    doc = Document(uri=os.path.join(directory, '../test_data/test_image.png'))
+    doc.convert_image_uri_to_blob()
+    img = Image.fromarray(doc.blob.astype('uint8'))
+    img = img.resize((96, 96))
+    img = np.array(img).astype('float32') / 255
+    doc.blob = img
+    assert doc.embedding is None
```

Comment:
I'm puzzled about what is happening here - shouldn't the first two lines be enough?